app/src/services/projects.ts

```diff
@@ -368,86 +368,122 @@ async function toastedProjectOperation(label: string, fn?: () => Promise<void>)
 }
 
 export async function createProject(project: SmartContractProjectCreate) {
-    await toastedProjectOperation('Create project', async () => {
-        const chainId = getProjectRegistryChainId()
+    await toastedProjectOperation('Create project', () => {
+        return new Promise(async (resolve, reject) => {
```

Comment:
Async executor is bad practice. [More about it.](https://eslint.org/docs/latest/rules/no-async-promise-executor)